### PR TITLE
Update to ember 2.10 family

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
 ---
 language: node_js
 node_js:
-  - "0.12"
+  - "4"
```
Comment (stefanpenner) [inline .travis.yml]: we should likely match ember-cli's here: https://github.com/ember-cli/ember-cli/blob/master/.travis.yml#L4-L8

**Comment by cibernox** [comment]
@stefanpenner Applied and travis green

**Comment by stefanpenner** [comment]
@cibernox do you mind adding the node versions ember-cli supports?

**Comment by cibernox** [comment]
@stefanpenner If you mean in the `travis.yml`, I already added 0.12. That is the minimum version now. You mean somewhere else?

**Comment by stefanpenner** [comment]
> @stefanpenner If you mean in the travis.yml, I already added 0.12. That is the minimum version now. You mean somewhere else?

it would be great if we could test all the versions we officially support, which i believe requires travis.config to contain:


```
  - "0.12"
  - "4"
  - "6"
  - "7"
```

**Comment by cibernox** [comment]
I just learned that you can test agains more than one version of node that way. I'm not much into node, I always assumed that testing agains the minimum version was enough to guarantee compatibility forward. Done

**Comment by stefanpenner** [comment]
@cibernox makes sense, this was we catch issues early. Rather then as annoyances to ours users.

**Pull request merged**